DemoApp/Sources/Components/Stats/ParticipantStatsModifier.swift

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import SwiftUI
+import StreamVideo
+
+struct ParticipantStatsModifier: ViewModifier {
+
+    @State private var presentStats = false
+
+    var call: Call?
+    var participant: CallParticipant
+
+    func body(content: Content) -> some View {
+        content
+            .overlay(
+                VStack {
+                    HStack {
+                        Spacer()
+                        contentView
+                    }
+
+                    Spacer()
+                }
+                .padding(6)
+            )
+    }
+
+    @ViewBuilder
+    private var contentView: some View {
+        if call != nil {
+            Button {
+                presentStats = true
+            } label: {
+                Image(systemName: "info")
+                    .foregroundColor(.white)
+                    .padding(9)
+                    .background(Color.black.opacity(0.6))
+                    .clipShape(Circle())
+                    .clipped()
+            }
+            .popover(isPresented: $presentStats) {
+                if let call {
+                    NavigationView {
+                        Group {
+                            GeometryReader { proxy in
+                                if proxy.frame(in: .global) != .zero {
```

Comment:
Why do we need this? Better to avoid if and use opacity instead.

Reply:
It's not needed anymore because of the minimum sizes. On iPad the GeometryReader was returning 0 on height for the first draw causing a crash